manifests/t/TeamIDE/TeamIDE/2.6.10/TeamIDE.TeamIDE.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TeamIDE.TeamIDE
+PackageVersion: 2.6.10
+InstallerType: nullsoft
+UpgradeBehavior: install

```

Comment:
```suggestion
InstallerType: nullsoft
InstallerSwitches:
  Upgrade: --updated
UpgradeBehavior: install
```

Reply:
@Exorcism0666 , add the above `InstallerSwitches` so your Manifest Metadata is complete when compared to Previous Versions, therefore satisfying https://github.com/microsoft/winget-pkgs/labels/Manifest-Metadata-Consistency Requirement.